modules/login/src/main/frontend/src/login/signUpForm.js

```diff
@@ -217,7 +217,7 @@ class SignUpForm extends React.Component {
           this.props.handleLogin && this.props.handleLogin(false);
           response.json().then((data) => {
             let errMsg = data?.error?.message;
-            errMsg = (errMsg ? errMsg : "Unknown Error");
+            errMsg = (errMsg ?? "Unknown Error");
```

Comment:
Another instance were previously we were replacing errMsg="" with "Unknown error" and now we're not. It might not matter in practice (didn't check all the data flow), but to preserve the behavior, use `||` instead of `??`.

```suggestion
            errMsg = (errMsg || "Unknown Error");
```